### ISSUE TITLE
local_subnet_cidr Update Fails on site2cloud Connection

### ISSUE THREAD

**Issue by deutmeyerbrianpfg** [AviatrixSystems/terraform-provider-aviatrix#1017]
**Describe the problem**

After building a aviatrix_site2cloud connection with a policy based tunnel using a mapped connection.  I needed to change the local_subnet_cidr to a new IP.  

Plan:
```
Terraform will perform the following actions:

  # aviatrix_site2cloud.this will be updated in-place
  ~ resource "aviatrix_site2cloud" "this" {
        id                         = "this_vpn~vpc-0123456789asd1g5r"
      ~ local_subnet_cidr          = "10.0.0.10/32" -> "10.0.2.10/32"
        # (21 unchanged attributes hidden)
    }
````
When the apply runs:
```
aviatrix_site2cloud.this: Modifying... [id=skygen_vpn~vpc-0123456789asd1g5r]

Error: failed to update Site2Cloud local_subnet_cidr: Rest API edit_site2cloud_conn Post failed: [AVXERR-SITE2CLOUD-0064] Local Subnet cannot be empty.

  on main.tf line 10, in resource "aviatrix_site2cloud" "this":
  10: resource "aviatrix_site2cloud" "this" {


Error: Terraform exited with code 1.
Error: Process completed with exit code 1.
```

**To Reproduce**

After building a site2cloud connection with a mapped setup, change the local_subnet_cidr to a different IP.

The local subnet cidr should update properly.

**Environment:**
 - Provider version = 2.19.1
 - Terraform version = 0.14.11
 - Aviatrix Controller version = UserConnect-6.4.2672

**Additional context**
What was the impact?

I can't change my configuration!

**Comment by deutmeyerbrianpfg** [comment]
Also tested provider 2.19.3 with Terraform 0.15.5, but the issue still exists.

**Comment by shuoliugit** [comment]
The issue was fixed in provider 2.19.5.
